src/cljs/org/broadinstitute/firecloud_ui/page/workspace/data/copy_data_entities.cljs

```diff
@@ -29,8 +29,11 @@
          (when (:server-error @state)
            [:div {:style {:marginTop "0.5em"}}
             [comps/ErrorViewer {:error (:server-error @state)}]])
-         [:div {:style {:marginTop "1em"}}
-          [comps/Button {:text "Copy"
+         [:div {:style {:display "inline-flex" :marginTop "1em"}}
+           [comps/Button {:text "Done"
```

Comment:
But wouldn't that be confusing? Because you can copy from a workspace multiple times and then hit "Done" to close the modal. If it said "Cancel" that might confuse the user into thinking that it might reverse what they did